packages/xo-web/src/xo-app/settings/remotes/remote.js

```diff
@@ -255,7 +255,7 @@ export default decorate([
                   className='form-control'
                   name='path'
                   onChange={effects.linkState}
-                  pattern='^(([^\\/]+)+(\\[^\\/]+)*)?$'
+                  pattern='^([^\\/]+(\\[^\\/]+)*)?$'
```

Comment:
Can you explain this change?

Reply:
I only removed the useless pair of parenthesis to match the other pattern.